### ISSUE TITLE
Problems with the report

### ISSUE THREAD

**Issue by JhonyVilla** [linkedin/qark#102]
Hello... When qark has finished to analyze my app I receive the next error

> ERROR - Problem with reporting; No html report generated. Please see the readme file for possible solutions.

I already have installed html5lib in my system, and i removed the folder report every time that i run qark, but it still failing.
I am using kali linux 2.0 x64bits

**Comment by sarveshkapre** [comment]
Hey,
Can you provide a sample app so that I can reproduce this issue. I am not sure whether this issue is "OS or app" specific.
Thanks.

**Comment by the-house** [comment]
Hi @sarveshkapre ,

Im having the same issue.  Please see below for error and apk in question.

Let me know if anymore info is needed.

Thanks!

![image](https://user-images.githubusercontent.com/19733056/30601328-ec30b394-9d1e-11e7-9e56-d23b3c5d9abb.png)

[testerAPK.zip](https://github.com/linkedin/qark/files/1314838/testerAPK.zip)

**Comment by theguly** [comment]
the same on OSX, a quick debug pointed out that the issue *could* be in report.py function write_manifest:
Error writing manifest: 'ascii' codec can't decode byte 0xd0 in position 1621: ordinal not in range(128)

**Comment by LinaQiu** [comment]
I had the same issue while using the tool. Below is the error message. I am on 10.12.6 macOS, 16GB. 
<img width="909" alt="screen shot 2017-09-27 at 10 37 22 am" src="https://user-images.githubusercontent.com/14211962/30919608-e5b2d64e-a36f-11e7-9da5-2b5848a4b7e7.png">

**Comment by taoy1** [comment]
I had the same issue. As you can see in qartMain.py, if common.reportInitSuccess == false, this error is generated.

The common.reportInitSuccess is set to true when the program begins, but in the write() function in report.py, line 249, `common.reportInitSuccess = False`, it could be set to false if there's any error.

You can put breakpoint in this line and see what happened. For me, the exception is caused by `pre_rendered_html.find("span", id=identity).append(new_span_tag)`. The findPending.py is calling this function with `identity="parsing error-issues-list"`, but this id does not exist in the html. The correct id should be "parsingerror-issues-list" (remove the space). After using the correct id, the report is generated successfully.

If this cannot solve your problem, you can use:

```
e = pre_rendered_html.find("span", id=identity)
if e is not None:
    e.append(new_span_tag)
```

**Comment by theguly** [comment]
looks ok my side

**Comment by neilatli** [comment]
Hi @JhonyVilla, @the-house, @theguly & @LinaQiu. 

This issue has been fixed with #111.  We also fixed some unrelated issues with the reporting in #133 so let us know if you have any further issues after updating.
